content/tutorials/quickstart/py-create-conditional-escrows.md

```diff
@@ -0,0 +1,625 @@
+---
+html: py-create-conditional-escrows.html
+parent: send-payments-using-python.html
+blurb: Create, finish, or cancel condition-based escrow transactions.
+labels:
+  - Accounts
+  - Quickstart
+  - Transaction Sending
+  - XRP
+---
+# Create Conditional Escrows Using Python
+
+This example shows how to:
+
+1. Create escrow payments that become available when any account enters a fulfillment code.
+
+2. Complete a conditional escrow transaction.
+
+3. Cancel a conditional escrow transaction.
+
+[![Conditional Escrow Tester Form](img/quickstart-py-conditional-escrow-1.png)](img/quickstart-py-conditional-escrow-1.png)
+
+
+## Prerequisites
+
+Download and expand the [Quickstart Samples](https://github.com/XRPLF/xrpl-dev-portal/tree/master/content/_code-samples/quickstart/py/){.github-code-download} archive.
+
+## Usage
+
+### Create Escrow
+
+You  create a condition-based escrow using a fulfillment code associated with a condition code. Create the condition/fulfillment pair using the `five-bells-condition` application.
+
+Install `five-bells-condition`:
+
+1. In a terminal window, navigate to your local `Quickstart` directory (for convenience).
+2. Enter the command `npm install five-bells-condition`.
```

Comment:
Ideally it would use a python library to show how this can all be done in python.